### PR TITLE
#69 Text Color 값 고정 컬러로 변경

### LINKED ISSUE #69: 카드 디테일이 빈 화면으로 보이는 이슈
- Color.Black / Color.White -> 고정 컬러로 변경 필요
- 카드 디테일 화면 외에도 Color.XXX 전부다 고정 컬러로 변경하기!!

### PR DESCRIPTION
**Pull request by seunghee63**
- https://github.com/Nexters/talkbbokki-android/issues/69

image
- 현재 다크모드! </br>
- 다른 뷰들은 모드 컬러 셋팅이 되어있네유
</br>
</br>
Closes #69

### CODE REVIEW

**Comment by Jiyoung9310** [comment]
Color.White / Color.Black으로 사용하지 말고 우리 고정컬러로 지정한 White / Black으로 사용하자
다른데에서도 Color.XXX로 되어있으면 고정 컬러로 수정 부탁해용

File: app/src/main/java/com/hammer/talkbbokki/presentation/bookmark/BookmarkListScreen.kt [hunk 1]
```diff
@@ -2,16 +2,7 @@ package com.hammer.talkbbokki.presentation.bookmark
...
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.PaddingValues
-import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.Spacer
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.height
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.*
```
Comment (seunghee63) [inline app/src/main/java/com/hammer/talkbbokki/presentation/bookmark/BookmarkListScreen.kt:5]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

File: app/src/main/java/com/hammer/talkbbokki/presentation/bookmark/BookmarkListScreen.kt [hunk 2]
```diff
@@ -21,28 +12,17 @@ import androidx.compose.material.Card
 import androidx.compose.material.ExperimentalMaterialApi
 import androidx.compose.material.Icon
 import androidx.compose.material.Text
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.collectAsState
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
-import androidx.compose.runtime.setValue
+import androidx.compose.runtime.*
```
Comment (seunghee63) [inline app/src/main/java/com/hammer/talkbbokki/presentation/bookmark/BookmarkListScreen.kt:15]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

File: app/src/main/java/com/hammer/talkbbokki/presentation/bookmark/BookmarkListScreen.kt [hunk 3]
```diff
@@ -21,28 +12,17 @@ import androidx.compose.material.Card
...
 import com.hammer.talkbbokki.R
 import com.hammer.talkbbokki.domain.model.TopicItem
 import com.hammer.talkbbokki.ui.dialog.BookmarkCancelDialog
-import com.hammer.talkbbokki.ui.theme.Black
-import com.hammer.talkbbokki.ui.theme.Gray04
-import com.hammer.talkbbokki.ui.theme.MainColor01
-import com.hammer.talkbbokki.ui.theme.MainColor02
-import com.hammer.talkbbokki.ui.theme.TalkbbokkiTypography
-import com.hammer.talkbbokki.ui.theme.White
+import com.hammer.talkbbokki.ui.theme.*
```
Comment (seunghee63) [inline app/src/main/java/com/hammer/talkbbokki/presentation/bookmark/BookmarkListScreen.kt:25]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

File: app/src/main/java/com/hammer/talkbbokki/presentation/suggestion/SuggestionScreen.kt [hunk 4]
```diff
@@ -34,12 +34,7 @@ import androidx.hilt.navigation.compose.hiltViewModel
 import com.hammer.talkbbokki.R
 import com.hammer.talkbbokki.ui.button.ButtonType
 import com.hammer.talkbbokki.ui.button.CommonLargeButton
-import com.hammer.talkbbokki.ui.theme.Gray04
-import com.hammer.talkbbokki.ui.theme.Gray06
-import com.hammer.talkbbokki.ui.theme.MainColor01
-import com.hammer.talkbbokki.ui.theme.MainColor02
-import com.hammer.talkbbokki.ui.theme.TalkbbokkiTypography
-import com.hammer.talkbbokki.ui.theme.White
+import com.hammer.talkbbokki.ui.theme.*
```
Comment (seunghee63) [inline app/src/main/java/com/hammer/talkbbokki/presentation/suggestion/SuggestionScreen.kt:37]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

File: app/src/main/java/com/hammer/talkbbokki/presentation/topics/TopicListScreen.kt [hunk 5]
```diff
@@ -35,9 +35,7 @@ import androidx.hilt.navigation.compose.hiltViewModel
 import com.hammer.talkbbokki.R
 import com.hammer.talkbbokki.domain.model.TopicItem
 import com.hammer.talkbbokki.presentation.showRewardedAd
-import com.hammer.talkbbokki.ui.theme.Gray07
-import com.hammer.talkbbokki.ui.theme.TalkbbokkiTypography
-import com.hammer.talkbbokki.ui.theme.White
+import com.hammer.talkbbokki.ui.theme.*
```
Comment (seunghee63) [inline app/src/main/java/com/hammer/talkbbokki/presentation/topics/TopicListScreen.kt:39]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

**Review by Jiyoung9310 — APPROVED** [review]

**Pull request merged**